### PR TITLE
Add helpers to decode multiple artists/genre (when a file has some)

### PR DESCRIPTION
**Pull request by martpie**
Fixes https://github.com/polyfloyd/rust-id3/issues/84

I guess this could be added to more tags, up to you. But that's all I need for now.

I'm a rust noob, so don't hesitate to tell me what can be improved in my code.

### CODE REVIEW

File: src/tag.rs [hunk 1]
```diff
@@ -769,4 +769,19 @@ mod tests {
         let lyrics = tag.synchronised_lyrics().next().unwrap();
         assert_eq!(lyrics.description, "Description");
     }
+
+    #[test]
+    fn test_multiple_tags() {
+        // Create temp file
+        let tmp = tempfile::NamedTempFile::new().unwrap();
+        fs::copy("testdata/multi-tags.mp3", &tmp).unwrap();
```
Comment (polyfloyd) [inline src/tag.rs:777]: Copying test data is only needed if we write to it. This test only reads, so could you just read from the testdata directly?

File: src/tag.rs [hunk 2]
```diff
@@ -769,4 +769,19 @@ mod tests {
         let lyrics = tag.synchronised_lyrics().next().unwrap();
         assert_eq!(lyrics.description, "Description");
     }
+
+    #[test]
+    fn test_multiple_tags() {
```
Comment (polyfloyd) [inline src/tag.rs:774]: Could you rename this to `test_issue_84`? Makes it easier to get some context about what was fixed

**Review by polyfloyd — CHANGES_REQUESTED** [review]

**Review by polyfloyd — CHANGES_REQUESTED** [review]
Could you get rid of that .DS_Store file?

**Comment by martpie** [comment]
Ah damn macOS 😒

**Comment by martpie** [comment]
Thank you! FYI, `audiotags` seems not maintained and I may fork it and improve it over time, so it's probably I'll open more issues/PRs in the future 👍

**Pull request merged**